themes/bootstrap3/templates/eds/advanced.phtml

```diff
@@ -22,9 +22,9 @@
           <option value="OR"><?=$this->transEsc("OR")?></option>
           <option value="NOT"><?=$this->transEsc("NOT")?></option>
         </select>
-        <input id="search_lookfor0_<?=$search ?>" name="lookfor0[]" class="adv-term-input form-control" type="text" value=""/>
+        <input id="search_lookfor0_<?=$search ?>" name="lookfor0[]" class="adv-term-input form-control" type="text" value="" aria-label="Search text <?=$search ?>"/>
         <span class="help-block hidden-xs"><?=$this->transEsc("in")?></span>
-        <select id="search_type0_<?=$search ?>" name="type0[]" class="adv-term-type form-control">
+        <select id="search_type0_<?=$search ?>" name="type0[]" class="adv-term-type form-control" aria-label="Search type <?=$search ?>">
```

Comment:
You can concatenate translations for phrases like this for now but they should be tokenized in the future.